substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/hub/AnnotationsEncoding.java

```diff
@@ -146,4 +148,10 @@ public static Object encodeAnnotations(Set<Annotation> allAnnotations, Set<Annot
 
         return new AnnotationsEncoding(encoding, position);
     }
+
+    public static void filterHostedAnnotations(Collection<Annotation> annotations) {
+        if (annotations != null) {
+            annotations.removeIf(annotation -> annotation.toString().contains("com.oracle.svm.core.annotate"));
```

Comment:
No, we cannot have literal package or class names like that. We need to check if the class of the annotation is in the current `Platform`.

Reply:
Sure. This was only a placeholder.